android/src/main/java/io/rownd/android/models/repos/AuthRepo.kt

```diff
@@ -134,22 +136,32 @@ class AuthRepo @Inject constructor(private val rowndContext: RowndContext) {
 
         refreshTokenJob = CoroutineScope(Dispatchers.IO).async {
             Log.d("Rownd.Auth", "Refreshing tokens: in progress")
+            val span = rowndContext.telemetry?.startSpan("refreshTokenAsync")
+            val refreshToken = stateRepo.state.value.auth.refreshToken ?: throw NoAccessTokenPresentException("No refresh token was available")
```

Comment:
Do you think this should be a separate type specific to refresh tokens, like NoRefreshTokenPresentException?

Reply:
might as well!